api/jest.config.js

```diff
@@ -27,5 +27,7 @@ module.exports = {
     "cobertura"
   ],
   testResultsProcessor: "jest-sonar-reporter",
-  testEnvironment: "node"
+  testEnvironment: "node",
+  resetMocks: true,
+  restoreMocks: true
```

Comment:
Hi Ian, just wondering if we want to add some comment here about what they are doing and why we need them? thanks!!

Reply:
Good idea, thanks Catherine, Information added.